### PR TITLE
Upgrade azure-mgmt-cosmosdb version to 0.2.1

### PR DESCRIPTION
**Pull request by dmakwana**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [X] Each command and parameter has a meaningful description.
- [X] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-cosmosdb/setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
```
Comment (troydai) [inline src/command_modules/azure-cli-cosmosdb/setup.py]: Update the version in `setup.py` as well.

**Comment by troydai** [comment]
Are these manual tested automated? If not can you automate them?

**Comment by moderakh** [comment]
@troydai the tests are not automated yet. I need to learn about test framework for azure-cli. I am hoping to  take another look at automating the tests (if possible) once I get a bit more time.

**Comment by troydai** [comment]
@moderakh sure. Please feel free to contact me if you have any question regarding the test framework. You can begin with this document: https://github.com/Azure/azure-cli/blob/dev/doc/scenario_base_tests.md

**Comment by derekbekoe** [comment]
CI is failing as you need to update setup.py

**Comment by dmakwana** [comment]
@moderakh I've tested these.
@derekbekoe I've updated the setup.py file

**Comment by troydai** [comment]
@moderakh please sign off.

**Comment by moderakh** [comment]
@troydai I signed off a while back. Doesn't github show my approval?

**Comment by troydai** [comment]
@moderakh I saw it. It is in the history but doesn't appear in the summary box. Good. Thank you.

**Pull request merged**